pgmpy/factors/continuous/LinearGaussianCPD.py

```diff
@@ -29,106 +31,56 @@ class LinearGaussianCPD(BaseFactor):
     p(Y |x) = N(β0 + β.T * x ; σ2)
```

Comment:
@ankurankan Can you modify the formula to use μ instead? Feels more relatable with mean.